### ISSUE TITLE
Need a mechanism to prevent further dependency updates until a PR is merged

### ISSUE THREAD

**Issue by hoyosjs** [dotnet/arcade#8672]
- [ ] This issue is blocking 
- [x] This issue is causing unreasonable pain

Often enough, in runtime a dependency update breaks things and codeflow stops for a while. Once things are in place you'd like to get that PR in to unblock people fast and propagate it up the stack, but maestro-bot often keeps pushing updates. This ends up retriggering CI with can take several hours. This is particularly bad in the case of batched updates from a large number of repos.

Possible ways of working around this now:
- Disable subscriptions or lower their frequency. Not as foolproof if there is a large number of subs.
- Disable batching - results in multiple commits that could've been put together.

Possible solutions:
- Add a label that tells maestro to avoid updates and not crate any new PRs.
- Does maestro update drafts? If so, that could be enough (hopefully without opening a new PR)

cc: @dotnet/runtime-infrastructure

**Comment by riarenas** [comment]
We are not planning on adding features to Maestro anytime soon, and we've hesitated in the past to add self-serve features that facilitate disabling dependency updates. 

Our recommendation here is to either continue disabling batching, with the knowledge this results in a separate PR, or if there's a lot of work that needs to go into a darc branch, to branch off it and do the work there.

We're going to close this for now.
